src/cartesian/ErrorBar.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileOverview Render a group of error bar
+*/
+import React, { Component, PropTypes } from 'react';
+import Layer from '../container/Layer';
+
+class ErrorBars extends Component {
```

Comment:
Not really sure the best way to change the domain of YAxis before adding error bars. Any thoughts?